CONTRIBUTING.md

```diff
@@ -1,11 +1,51 @@
-# How to Contribute
+# Contributing to graphql-kimera
+
+> Please note that this project is released with a [Contributor Code of Conduct](./CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
 
 Kimera is one of [Lola Tech's](https://www.lola.tech) first open source project that is both under very active development and is also being used to ship code to our enterprise customers.
 
-## Code of Conduct
+## Found an Issue?
+
+Thank you for reporting any issues you find. We do our best to test and make graphql-kimera as solid as possible, but any reported issue is a real help.
+
+> When raising an issue please use the template provided as a guideline, provided to you.
+
+## Want a new feature?
+
+> For raising a feature request please use the template provided as a guideline, provided to you.
+
+## Branch Organization
+
+Submit all changes directly to the `master` branch. We don’t use separate branches for development or for upcoming releases. We do our best to keep master in good shape, with all tests passing.
+
+Code that lands in master must be compatible with the latest stable release. It may contain additional features, but no breaking changes. We should be able to release a new minor version from the tip of master at any time.
+
+## Code structure
+
+Currently the project is managed with lerna and it contains 2 packages:
+
+- [graphql-kimera](https://github.com/lola-tech/graphql-kimera/tree/master/packages/graphql-kimera): containing the source code and tests for the module published on npm.
+- [graphql-kimera-docs](https://github.com/lola-tech/graphql-kimera/tree/master/packages/graphql-kimera-docs): a [docusaurus](https://v2.docusaurus.io/) containing the docusaurus app for managing and changing the documentation.
```

Comment:
~a [docusaurus](https://v2.docusaurus.io/) containing the docusaurus app for managing and changing the documentation.~
a [docusaurus app](https://v2.docusaurus.io/) for managing the documentation.

Reply:
: a docosaurus app [link] for managing the documentation

Reply:
Can we stick to `a docosaurus app for managing the documentatio`? The link appears before this text.